### ISSUE TITLE
Time Series Naming using /x13/requests

### ISSUE THREAD

**Issue by srothmei** [nbbrd/jdemetra-ws#10]
The following problem occurs to me whilst using the method /x13/requests:
The timeseries used for Seasonal Adjustment get's a (fixed) generic naming inside the resulting XML-File, namely "series1".

I'm inputting a x13requests-file generated by the JWSRequests-tool. Inside this file everything looks normal i.e. the naming is correct:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<x13:X13Requests xmlns:tss="ec/eurostat/jdemetra/core" xmlns:x13="ec/eurostat/jdemetra/sa/x13" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:modelling="ec/eurostat/jdemetra/modelling" xmlns:sa="ec/eurostat/jdemetra/sa" xmlns="ec/tss.core" xmlns:bench="ec/eurostat/jdemetra/benchmarking">
    <x13:Item>
        <x13:Series name="nominal&#xA;OW45_01nom [frozen]">
[Omited Some Code here for the sake of better readability]
        </x13:Series>
    </x13:Item>
    <x13:Context/>
</x13:X13Requests>
```

Once i get the results back from the Webservice it looks like this:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<tss:InformationSet xmlns:tss="ec/eurostat/jdemetra/core" xmlns:xs="http://www.w3.org/2001/XMLSchema">
    <tss:item name="series1">
        <tss:Subset>
            <tss:item name="decomposition">
[Omited Some Code here for the sake of better readability]
            </tss:item>
            <tss:item name="m-statistics">
[Omited Some Code here for the sake of better readability]
            </tss:item>
        </tss:Subset>
    </tss:item>
</tss:InformationSet>
```

I looked through my input-material but the naming "series1" doesnt occur at all so i'm assuming something went wrong after i fired the WS-request?

**Comment by maggima** [comment]
Hello,

Thanks for reporting this issue.
There's nothing wrong about the way you used the service. During the processing, only the data are loaded into the results so it needs to be adapted in the `X13XmlProcessor` and `TramoSeatsXmlProcessor`. The classes are located in jdemetra-core so it will only be changed in the next release of the core libraries (2.2.1) early 2018.

Regards,

Mats

**Comment by maggima** [comment]
Hello @srothmei,

Release candidate 2 of the webservice should fix your issue about names.
The fix is in v2.2.1-RC2 of JDemetra core libraries.

Here is a link to the commit : https://github.com/jdemetra/jdemetra-core/commit/dc74d0ee608db2d6088911677c9a23f1def5e522

Best regards,

Mats
